integration_tests/_helpers.py

```diff
@@ -130,6 +130,13 @@ def email(self) -> _Email:
     def username(self) -> Optional[_Username]:
         return self.profile.username
 
+    def gql(
+        self,
+        query: str,
+        variables: Optional[Mapping[str, Any]] = None,
+    ) -> Dict[str, Any]:
+        return _gql(self, query=query, variables=variables)
```

Comment:
We probably need a `raise_for_status` inside `_gql`.

Reply:
it actually delegates down to httpx client (as all helpers do), and the raising is done in the json parsing step (for all the helpers)